modules/host/contractmanager/writeaheadlogsync.go

```diff
@@ -178,7 +189,7 @@ func (wal *writeAheadLog) commit() {
 		if err != nil {
 			wal.cm.log.Severe("ERROR: unable to create write-ahead-log:", err)
 		}
-		// Write the metadata into the WAL.
+		// Write the mmetaetadata into the WAL.
```

Comment:
I don't know if some combination of find-and-replace was involved, but this is the funniest typo I've seen in a while.